dashboard/db/migrate/20170505231542_remove_name_from_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNameFromPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :plc_courses, :name
```

Comment:
Good point. No need to fix this now anyway since it's already deployed. It's just something to consider for future migrations. Another option is a no-op down migration, like [here](https://github.com/code-dot-org/code-dot-org/blob/9cd7b239997767723b6e1a728b5890f96d0cb93c/dashboard/db/migrate/20170428002439_create_courses.rb#L31).